ui/app/components/namespace-picker.hbs

```diff
@@ -12,11 +12,20 @@
       <Hds::Form::TextInput::Field
         @value={{this.searchInput}}
         @type="search"
+        aria-label="Search namespaces"
         placeholder="Search"
+        {{on "keydown" this.onKeyDown}}
         {{on "input" this.onSearchInput}}
+        {{did-insert this.focusSearchInput}}
```

Comment:
I'm guessing that the focus on opening this was previous behavior? If so, I can't say we always did focus things according to best a11y practices. Looking at the hds pattens on[ focus](https://hds-website-hashicorp.vercel.app/foundations/accessibility#focus-management), it might be good to review that we can indeed follow them.

If this was old behavior, maybe we should revisit this with Design's input (not blocking here, but come back with a ticket after discussion)? What are your thoughts?